TurkishFilter/sections/specific.txt

```diff
@@ -41,6 +41,7 @@ yemektarifleri.org,lifebursa.com,mshowto.org,macizlee1.com,macizlee.ne,dunyaatla
 ! ###adStarter
 fullhdfilmcibaba.com,720pizlemek.com,1080pfilmizle.net,fullhdfilmcidayi.com,hdfilmizledayi.com,720-pizleme.com,fullhdfilmizlebaba.com,evrenselfilmlerim.net###adStarter
 !
+forum.mevsim.org##.xgt-konteyner
```

Comment:
Why not `forum.mevsim.org##div[class^="xgt-Banner"]`?

Reply:
Because the music at the bottom blocks the listening place

Reply:
Ok, improved the rule.